library/core/src/slice/mod.rs

```diff
@@ -384,6 +385,7 @@ impl<T> [T] {
     #[inline]
     #[stable(feature = "slice_first_last_chunk", since = "1.77.0")]
     #[rustc_const_stable(feature = "slice_first_last_chunk", since = "1.77.0")]
+    #[cfg_attr(bootstrap, rustc_allow_const_fn_unstable(const_mut_refs))]
```

Comment:
Why did you add this attribute? This function was already stable before...

Happens again below.

Reply:
Since const_mut_refs is a compiler feature the bootstrap build needs to enable the feature (since the current beta compiler doesn't have it stabilized)

Error if removed:
```
   Compiling std v0.0.0 (/home/chrisi/Documents/code/rust/rust/library/std)
error: const-stable function cannot use `#[feature(const_mut_refs)]`
   --> core/src/slice/mod.rs:358:50
    |
358 |     pub const fn first_chunk_mut<const N: usize>(&mut self) -> Option<&mut [T; N]> {
    |                                                  ^^^^^^^^^
    |
help: if the function is not (yet) meant to be stable, make this function unstably const
    |
358 +     #[rustc_const_unstable(feature = "...", issue = "...")]
359 |     pub const fn first_chunk_mut<const N: usize>(&mut self) -> Option<&mut [T; N]> {
    |
help: otherwise, as a last resort `#[rustc_allow_const_fn_unstable]` can be used to bypass stability checks (but requires team approval)
    |
358 +     #[rustc_allow_const_fn_unstable(const_mut_refs)]
359 |     pub const fn first_chunk_mut<const N: usize>(&mut self) -> Option<&mut [T; N]> {
    |

error: const-stable function cannot use `#[feature(const_mut_refs)]`
   --> core/src/slice/mod.rs:365:27
    |
365 |             Some(unsafe { &mut *(self.as_mut_ptr().cast::<[T; N]>()) })
    |                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    |
help: if the function is not (yet) meant to be stable, make this function unstably const
    |
358 +     #[rustc_const_unstable(feature = "...", issue = "...")]
359 |     pub const fn first_chunk_mut<const N: usize>(&mut self) -> Option<&mut [T; N]> {
    |
help: otherwise, as a last resort `#[rustc_allow_const_fn_unstable]` can be used to bypass stability checks (but requires team approval)
    |
358 +     #[rustc_allow_const_fn_unstable(const_mut_refs)]
359 |     pub const fn first_chunk_mut<const N: usize>(&mut self) -> Option<&mut [T; N]> {
    |
```

Reply:
Yes, `first_chunk_mut` needs this feature. But `first_chunk` does not.

Reply:
Ah right sorry. Yeah I don't know why I added this. Removing it.